### PR TITLE
Update the README with setup commands

### PR DESCRIPTION
**Pull request by losvedir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+$ source env/bin/activate
+$ pip3 install --upgrade pip
+$ pip3 install --upgrade jupyter matplotlib numpy pandas scipy scikit-learn
+```
+
+Test that it worked. The following should run with no output and no errors.
+
+```
+$ python3 -c "import jupyter, matplotlib, numpy, pandas, scipy, sklearn"
+```
```
Comment (lemald) [inline README.md:31]: Maybe include a quick note at the end to run `jupyter notebook` after `source env/bin/activate` to actually open the notebook.

Comment (losvedir) [reply README.md:31]: Ah, good idea.

Comment (losvedir) [reply README.md:31]: Done.

**Review by lemald — APPROVED** [review]
🤖🏫👍

**Pull request merged**